### PR TITLE
MEN-3852: rename column retries to attempts in deployment device list

### PR DESCRIPTION
**Pull request by tranchitella**
This commit also updates the content of the table to make it consistent
with the new label.

Signed-off-by: Fabio Tranchitella <fabio.tranchitella@northern.tech>

### CODE REVIEW

**Review by mzedel — APPROVED** [review]

**Comment by tranchitella** [comment]
@michaelatmender as this is an UX task, I'm waiting for your approval before merging

**Review by michaelatmender — APPROVED** [review]
Thanks, this is a good short term fix. I may follow up with more adjustments later, on the deployment creation view.

**Pull request merged**
